packages/sx05re/emuelec/config/emuelec/ports/SDLPoP.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2020-present Shanti Gilbert (https://github.com/shantigilbert)
+
+# Source predefined functions and variables
+. /etc/profile
+
+JSLISTENCONF="/emuelec/configs/jslisten.cfg"
```

Comment:
Can you check how ports are done now, 

https://github.com/EmuELEC/EmuELEC/blob/6d4e92b4852d1bfc013ee1f09ae9ea197b32c989/packages/sx05re/emuelec/config/emuelec/ports/Diablo.sh

and change accordingly

Reply:
and done